soccer/vision/filter/KalmanFilter.cpp

```diff
@@ -0,0 +1,32 @@
+#include "KalmanFilter.hpp"
```

Comment:
I have been informed that there is a wikipedia page with most of this information. It would be nice if you at least added the link